travis.sh

```diff
@@ -67,6 +67,59 @@ build_java_oracle8() {
   build_java
 }
 
+build_android() {
```

Comment:
Makes sense.

[this bug](https://github.com/travis-ci/travis-ci/issues/3534) seems to think you can use a local cache to further speed things up just by adding some config tweaks. Might be worth it?